logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/SendKnockUseCase.kt

```diff
@@ -78,16 +74,11 @@ class SendKnockUseCase internal constructor(
                 isSelfMessage = true
             )
             persistMessage(message)
-                .flatMap {
-                    messageSender.sendMessage(message)
-                }
-                .onSuccess {
-                    messageRepository.updateMessageStatus(MessageEntity.Status.SENT, conversationId, generatedMessageUuid)
```

Comment:
This is already done in `MessageSender` (`promoteMessageToSentUpdatingServerTime`)